.github/workflows/close-stale-issues.yaml

```diff
@@ -0,0 +1,29 @@
+---
+jobs:
+  stale:
+    runs-on: 'ubuntu-latest'
+    steps:
+    - uses: 'actions/stale@v9'
+      with:
+        days-before-close: '15'
+        days-before-stale: '45'
```

Comment:
Should we update this to smaller values for the testing period?

Reply:
ah yes, I meant to change these